doc/api/http2.md

```diff
@@ -540,16 +540,28 @@ All other interactions will be routed directly to the socket.
 added: v8.4.0
 -->
 
+Provides miscellaneous information about the current state of the
+`Http2Session`.
+
 * Value: {Object}
-  * `effectiveLocalWindowSize` {number}
-  * `effectiveRecvDataLength` {number}
-  * `nextStreamID` {number}
-  * `localWindowSize` {number}
-  * `lastProcStreamID` {number}
-  * `remoteWindowSize` {number}
-  * `outboundQueueSize` {number}
-  * `deflateDynamicTableSize` {number}
-  * `inflateDynamicTableSize` {number}
+  * `effectiveLocalWindowSize` {number} The current local (receive)
+    flow control window size for the `Http2Session`.
+  * `effectiveRecvDataLength` {number} The current number of bytes
+    that have been received since the last flow control `WINDOW_UPDATE`.
+  * `nextStreamID` {number} The numeric identifier to be used the
+    next time a new `Http2Stream` is created by this `Http2Session`.
+  * `localWindowSize` {number} The number of bytes that the remote peer can
+    send without receiving a `WINDOW_UPDATE`.
+  * `lastProcStreamID` {number} The numeric id of the `Http2Stream`
+    for which a `HEADERS` or `DATA` frame was most recently received.
+  * `remoteWindowSize` {number} The number of bytes that this `Http2Session`
+    may sent without receiving a `WINDOW_UPDATE`.
```

Comment:
A nit: sent -> send

Reply:
Fixed! btw, @vsemozhetbyt ... I haven't said this enough, I *really* appreciate that you review the doc changes so closely and catch these typos. I tend to completely overlook them so I'm very happy to have someone paying such close attention to them.

Reply:
Thank you) I was often embarrassed to make only these petty comments about the nits while you folks do all the heavy lifting and all the great changes) I am glad this was not so annoying as a thought)